pkg/meta/dynamodb/dynamodb.go

```diff
@@ -98,814 +104,711 @@ func (dwr *DynamoDB) SetImageTrustStore(imgTrustStore mTypes.ImageTrustStore) {
 	dwr.imgTrustStore = imgTrustStore
 }
 
-func (dwr *DynamoDB) SetManifestData(manifestDigest godigest.Digest, manifestData mTypes.ManifestData) error {
-	mdAttributeValue, err := attributevalue.Marshal(manifestData)
+func (dwr *DynamoDB) SetProtoImageData(digest godigest.Digest, protoImageData *proto_go.ImageData) error {
```

Comment:
Just rename the function.

Reply:
I can't rename this function. It's names as what it's supposed to do. There is also a SetImageData that is part of the MetaDB interface

Reply:
Do we need a separate method taking `*proto_go.ImageData` as a parameter?
I thought in the final version we would have only SetImageData with the signature as in the interface.

And if we do, why is this method exported?